servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpPayloadWriter.java

```diff
@@ -27,24 +27,52 @@
  */
 public interface HttpPayloadWriter<T> extends PayloadWriter<T>, TrailersHolder {
 
+    /**
+     * {@inheritDoc}
```

Comment:
Does anyone know what's wrong with JDK21? Its output:
```
/home/runner/work/servicetalk/servicetalk/servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpPayloadWriter.java:36: warning: @inheritDoc used but trailers() does not override or implement any method.
    HttpHeaders trailers();
                ^
error: warnings found and -Werror specified
1 error
15 warnings

> Task :servicetalk-http-api:javadoc FAILED
```

JDK 8, 11, and 17 are happy with this javadoc.

Reply:
For not I just removed `@inheritDoc` and kept only new note. Should I copy the original javadoc from parent interface for all methods?